drivers/spi/spi_ll_stm32.c

```diff
@@ -683,6 +758,19 @@ static int transceive(const struct device *dev,
 		spi_context_buffers_setup(&data->ctx, tx_bufs, rx_bufs, 2);
 	}
 
+#if DT_HAS_COMPAT_STATUS_OKAY(st_stm32h7_spi)
+	if (cfg->fifo_enabled && SPI_OP_MODE_GET(config->operation) == SPI_OP_MODE_MASTER) {
```

Comment:
```suggestion
	if (SPI_OP_MODE_GET(config->operation) == SPI_OP_MODE_MASTER) {
```

to fix hardware CS (_#65802_) when fifo is not used.